network/uRadMonitor.30s.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
+# This script is used with BitBar for uRad Monitor
+
+# <bitbar.title>uRadMonitor view</bitbar.title>
+# <bitbar.version>v1.1</bitbar.version>
+# <bitbar.author>Martin LBB</bitbar.author>
+# <bitbar.author.github>martinlbb</bitbar.author.github>
+# <bitbar.desc>Extract radiation value from uRadMonitor and show it.</bitbar.desc>
+# <bitbar.dependencies>curl,grep,egrep,sed, bc</bitbar.dependencies>
+
+IP="192.168.0.251"
+
+#Script begin here
+#Retrieve value from uRad device
+uradhtml=$(curl -s http://$IP)
+
+
+#Determine detector and ratio for uSv
+if echo $uradhtml | grep -q "SBM20"; then
+  ratio=0.006315
+  detector="SBM20"
+elif echo $uradhtml | grep -q "SI29BG"; then
+  ratio=0.010000
+  detector="SI29BG"
+elif echo $uradhtml | grep -q "SBM19"; then
+  ratio=0.001500
+  detector="SBM19"
+elif echo $uradhtml | grep -q "STS5"; then
+  ratio=0.006666
+  detector="STS5"
+elif echo $uradhtml | grep -q "SI22G"; then
+  ratio=0.001714
+  detector="SI22G"
+elif echo $uradhtml | grep -q "SI3BG"; then
+  ratio=0.631578
+  detector="SI3BG"
+elif echo $uradhtml | grep -q "SBM21"; then
+  ratio=0.048000
+  detector="SBM21"
+elif echo $uradhtml | grep -q "LND712"; then
+  ratio=0.005940
+  detector="LND712"
+elif echo $uradhtml | grep -q "SBT9"; then
+  ratio=0.010900
+  detector="SBT9"
+elif echo $uradhtml | grep -q "SI1G"; then
+  ratio=0.006000
+  detector="SI1G"
+fi
+
+#Get latest CPM
+cpm=$(echo $uradhtml | grep -o "radiation:[0-9]*CPM" | egrep -o '[0-9]{1,9}')
+
+#Get average CPM
+avgcpm=$(echo $uradhtml | grep -o "average:[0-9].*CPM" | egrep -o "[0-9]{1,9}.[0-9]{2}")
+
+#Get Temperature (Degree Only today)
+temp=$(echo $uradhtml | grep -o "temperature:[0-9].*C" | egrep -o "[0-9]{1,3}.[0-9]{2}")
```

Comment:
```
temp=$(echo $uradhtml | grep -o "temperature:[0-9].*C" | egrep -o "[0-9]{1,3}.[0-9]{2}")
            ^-- SC2086: Double quote to prevent globbing and word splitting.
```